tools/f18/f18.cc

```diff
@@ -215,6 +216,13 @@ std::string CompileFortran(
     }
     Fortran::semantics::ResolveNames(Fortran::semantics::Scope::globalScope,
         parseTree, parsing.cooked(), directories);
+    const auto& Cook = parsing.cooked();
```

Comment:
For consistency: `const auto &cook{parsing.cooked()};`

We write '&' and '*' next to the identifier; we name variables, constants, and data members with lower-case initial letters; and we use braced initialization now that C++17 has fixed it to work with `auto`.

Reply:
done